mipengine/controller/controller_logger.py

```diff
@@ -1,42 +1,15 @@
 import logging
-import sys
-import time
-from functools import wraps
-import quart.logging as ctrl_logging
-from mipengine.controller import config as ctrl_config
 
 
-def getLogger(name):
-    logger = ctrl_logging.getLogger(name)
-    formatter = ctrl_logging.Formatter(
-        "%(asctime)s - "
-        "%(levelname)s - "
-        "CONTROLLER - CONTROLLER - CONTROLLER - "
-        "%(name)s - "
-        "%(funcName)s(%(lineno)d) - "
-        "%(message)s"
-    )
+def getRequestLogger():
+    """
+    Used for logging information produced after an endpoint request.
+    """
+    return logging.getLogger("controller_context")
 
-    handler = ctrl_logging.StreamHandler(sys.stdout)
-    logger.setLevel(ctrl_config.log_level)
-    handler.setFormatter(formatter)
-    logger.addHandler(handler)
 
-    return logger
-
-
-def log_method_call(func):
-    @wraps(func)
-    def wrapper(*args, **kwargs):
-        logger = getLogger(func.__name__)
-        starting_timestamp = time.time()
-        logger.info(f"*********** {func.__name__} method started ***********")
-        output = func(*args, **kwargs)
-        finish_timestamp = time.time()
-        tsm_diff = finish_timestamp - starting_timestamp
-        logger.info(
-            f"*********** {func.__name__} method succeeded in {tsm_diff} ***********"
-        )
-        return output
-
-    return wrapper
+def getBackgroundServiceLogger():
+    """
+    Used for logging information produced by any background service.
+    """
+    return logging.getLogger("controller_background_service")
```

Comment:
It is not obvious to me why there have to be 2 functions to get a logger. Why must there be something called a RequestLogger and something different called BackgroundServiceLogger?